src/Middleware/Diagnostics/test/UnitTests/ExceptionHandlerTest.cs

```diff
@@ -500,9 +500,14 @@ public async Task ExceptionHandlerNotFound_RethrowsOriginalError()
                                 httpContext.Response.StatusCode = StatusCodes.Status500InternalServerError;
                             }
 
-                            // The original exception is thrown
+                            // Invalid operation exception
                             Assert.NotNull(exception);
-                            Assert.Equal("Something bad happened.", exception.Message);
+                            Assert.Equal($"No exception handler was found, see inner exception for details of original exception. If an exception should not be thrown for 404 responses, set {nameof(ExceptionHandlerOptions.AllowStatusCode404Response)} to true.", exception.Message);
```

Comment:
Inline the interpolation for the test.
```suggestion
                            Assert.Equal("No exception handler was found, see inner exception for details of original exception. If an exception should not be thrown for 404 responses, set AllowStatusCode404Response to true.", exception.Message);
```